### PR TITLE
threatq: add user configurable page size

### LINKED ISSUE #9921: [ThreatQuotient] Enable configurable page_size
## Description

Update the ThreatQuotient module to allow user-configurable page_size

### Background

A customer requests an update that allows users to configure the `url.param.limit` via a `page_size` variable. 
A non-configurable hardcoded limit of 100 may be insufficient.

### PR DESCRIPTION
**Pull request by adrianchen-es**
## Proposed commit message

See title.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- Closes #9922

### CODE REVIEW

File: packages/ti_threatq/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.26.0"
+  changes:
+    - description: Support user configurable page size.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/9921
```
Comment (efd6) [inline packages/ti_threatq/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/9922
```

**Comment by efd6** [comment]
Why is this in the PR description?

> Note: At this point, this is too late, but I think it may have been a better initial choice to just expose the oauth config tree as a YAML field.

**Comment by adrianchen-es** [comment]
> Why is this in the PR description?
> 
> > Note: At this point, this is too late, but I think it may have been a better initial choice to just expose the oauth config tree as a YAML field.

That was incorrectly leftover from a copy and paste.

File: packages/ti_threatq/data_stream/threat/manifest.yml [hunk 2]
```diff
@@ -81,6 +81,14 @@ streams:
         default: "90d"
         description: >-
           Enforces all IOCs to expire after this duration. This setting is required to avoid "orphaned" IOCs that never expire. Use [Elasticsearch time units](https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html#time-units) in days, hours, or minutes (e.g 10d)
+      - name: page_size
+        type: integer
+        multi: false
+        required: false
+        default: "100"
```
Comment (efd6) [inline packages/ti_threatq/data_stream/threat/manifest.yml:88]:
```suggestion
        type: integer
        multi: false
        required: false
        default: 100
```

**Review by efd6 — DISMISSED** [review]

**Review by efd6 — APPROVED** [review]

**Pull request merged**